core/cli/serialization.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"encoding/json"
+	"io"
+	"time"
+
+	"go.polydawn.net/repeatr/api/def"
+
+	"github.com/inconshreveable/log15"
+	"github.com/ugorji/go/codec"
+)
+
+var _ io.Writer = &journalSerializer{}
+
+type ctxPairs []interface{}
+
+func (ctxPairs) MapBySlice() {}
+
+type logItem struct {
+	Level int       `json:"level"`
+	Msg   string    `json:"message"`
```

Comment:
I like the terser `msg` as the serialization key for how often this happens.  Possibly `lvl` along the same reasoning.  Going hog wild, maybe even `t` for time.